### ISSUE TITLE
Heatwave should author Build event Targets not Properties

### ISSUE THREAD

**Issue by chrpai** [firegiant/HeatWaveSupport#55]
### HeatWave Version

1.0.2

### Visual Studio Version

17.8.3

### Repro Steps

The author a prebuild and postbuild event that contains $(TargetPath).  Look at the .wixproj you'll see it authored properties but and the TargetPath will fail to evaluate.  Create a .NET SDK console application and do the same.  You'll see it authored Targets not Properties.  Transpose those targets to your .wixproj and it will work.

In a nutshell, Heatwave should be authoring Targets not Properties for build events.    See #44 #46.

### Actual Result

TargetDir fails to eval and the build fails.

### Expected Result

TargetDir should eval and the build should succeed.

**Comment by FireGiantHelp** [comment]
Duplicate of #44
